src/yang/ietf-bgp.yang

```diff
@@ -858,6 +858,54 @@ module ietf-bgp {
             "Clear neighbors action.";
 
           input {
+	    choice operation {
+	      default operation-admin;
+	      description
+	        "The type of operation for the clear action.";
+	      case operation-admin {
+	        leaf admin {
+		  type empty;
+		  description
+		    "Closes the Established BGP session with a BGP
+		     NOTIFICATION message with the Administrative
+		     Reset error subcode.";
+		  reference
+		    "RFC 4486.";
```

Comment:
In addition to citing the RFC number, I usually provide the title of the RFC. The format for the reference is:

"RFC XXXX: <Title of the RFC>, <Optionally, the word Section followed by the Section Number>.";

Reply:
I've added the RFC title.  No section is added since the entire RFC is about those codes.

Reply:
Ok with RFC